src/portal/src/index.html

```diff
@@ -4,7 +4,7 @@
 <head>
     <meta charset="utf-8">
     <title>Harbor</title>
-    <base href="/">
+    <base href="/test">
```

Comment:
Why is this set to `/test` now?

Reply:
@lindhe Thanks for the info. I forgot to change it back。 It should be "/"

Reply:
That's alright, it's easy to make mistakes. Even the code review from @wy65701436 didn't catch it. Please refer to this PR when you submit the fix, so it's easy to find for people looking at this comment thread. :)

Pro tip: Instead of using `git add .` to add all changes, you can use `git add -p` which lets you inspect each hunk of patch individually before adding.

Reply:
And maybe there should be a regression test implemented to catch this in the future?

Reply:
@lindhe Of course, Considering adding a new nightly(e2e) case to cover this

Reply:
Thank you for fixing this so quickly. That's nice.

I've noticed that PR descriptions are often very short in this project. I think that if you spent some more time writing a description of what this change is supposed to do, and perhaps mention any surprising work-arounds made in the code, it would be easier for the code reviewer to know if the code looks like it should. Just a thought! :)

Again, thank you for fixing this. I appreciate seeing that Harbor is actively worked on.